tools/functional-tester/etcd-runner/command/election_command.go

```diff
@@ -65,8 +68,8 @@ func runElectionFunc(cmd *cobra.Command, args []string) {
 				break
 			}
 		}
-		e := concurrency.NewElection(s, "electors")
 
+		e := concurrency.NewElection(s, elector)
```

Comment:
yep, when there are 3 elections running with the same elector string. it cause conflict with each other. the random string just make it rarely to happen. Maybe I should use pass in as a command arg? or something else to guarantee that elector string is unique.